pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting.md

```diff
@@ -0,0 +1,275 @@
+---
+$title: Formatting guides & tutorials 
+$order: 3
+description: 'File formatting requirements for amp.dev'
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+author: CrystalOnScript
+---
+
+Guides and tutorials are submitted in [Markdown](https://www.markdownguide.org/), with an additional frontmatter and shortcode formatting. 
+
+
+## Documentation locations
+
+Content on amp.dev is pulled from two repositories, amp.dev and AMPHTML. All reference documentation under components is pulled from AMPHTML, either from builtins or extensions. 
+
+
+
+*   [Built-in components ](https://github.com/ampproject/amphtml/tree/master/builtins)
+*   [Components](https://github.com/ampproject/amphtml/tree/master/extensions) 
+*   [Courses](https://github.com/ampproject/amp.dev/tree/future/pages/content/amp-dev/documentation/courses) 
+*   [Examples](https://github.com/ampproject/amp.dev/tree/future/pages/content/amp-dev/documentation/examples)
+*   [Guides & tutorials](https://github.com/ampproject/amp.dev/tree/future/pages/content/amp-dev/documentation/guides-and-tutorials)
+
+There are several other documents that are imported to amp.dev from AMPHTML. They are [listed in this file](https://github.com/ampproject/amp.dev/blob/future/platform/config/imports/spec.json)._ If you update any of these documents in the amp.dev repository it will be re-written on the next build_.  
```

Comment:
Maybe change "If you update any of these documents in the amp.dev repository it will be re-written on the next build" to be more direct, e.g. "Don't update these documents in the amp.dev repository – your changes will get overwritten on subsequent builds!"